src/kemal/config.cr

```diff
@@ -12,7 +12,7 @@ module Kemal
     HANDLERS        = [] of HTTP::Handler
     CUSTOM_HANDLERS = [] of Tuple(Nil | Int32, HTTP::Handler)
     FILTER_HANDLERS = [] of HTTP::Handler
-    ERROR_HANDLERS  = {} of Int32 => HTTP::Server::Context, Exception -> String
+    ERROR_HANDLERS  = {} of (Int32 | Exception.class) => HTTP::Server::Context, Exception -> String
```

Comment:
**thought:** I'm wondering if it makes sense to mash up `Int32` and `Exception.class` keys in a single hash.
Might be better to have two separate collections. They're used completely independent of each other. There's no need to expose both under the same name `Kemal.error_handlers`.
Maybe we could introduce a new name `Kemal.exception_handlers` for this feature?